stubs/django/urls/conf.pyi

```diff
@@ -0,0 +1,8 @@
+from typing import Any, Mapping, Optional, Union
+
+from django.urls.resolvers import URLPattern, URLResolver
```

Comment:
Similar to my comment about the `Model` errors above, the simple workaround here is to change the import to: `from django.urls import URLPattern, URLResolver`